fe1-web/src/features/digital-cash/reducer/DigitalCashReducer.ts

```diff
@@ -28,7 +28,7 @@ export interface DigitalCashReducerState {
 
   /**
    * A mapping between public key hashes and an array of hash of transactions which contains
-   * this public key hash in one or more of their TxOuts
+   * this public key hash in one or more of their TxOuts or their TxIns
```

Comment:
Does this not create issues with `requestSendTransaction` that uses the transactions returned by `DigitalCashStore.getTransactionsByPublicKey` (which is `laoState.transactionsByPubHash`) as an input to a new posted transaction?

It seems as previously `transactionsByPubHash` was used as a way to refer to previous transaction storing the user's balance when a new transaction is posted, all of the funds are accessed where one part is transferred to the beneficiary and the remainder to the user themselves. I think your changes break this but it might very well be that I am missing a change you did to prevent this from happening?

Reply:
I think you're right, I should not be using this. I checked if this was used anywhere but forgot to go through the DigitalCashStore 😅.
I'll create another record in the reducer to store this.

Reply:
Nice catch, I completely missed this.